File: 3476-find-minimum-operations-to-make-all-elements-divisible-by-three/find-minimum-operations-to-make-all-elements-divisible-by-three.cpp
class Solution {
public:
    int minimumOperations(vector<int>& nums) {
        int minOpr = 0 ;
        for(int i = 0 ; i<nums.size() ; i++){
           
            if(nums[i] %3 != 0)
                minOpr++ ;
        }

        return minOpr ;
    }
};

File: 3476-find-minimum-operations-to-make-all-elements-divisible-by-three/find-minimum-operations-to-make-all-elements-divisible-by-three.java
class Solution {
    int solve(int[]arr , int i){
        if(i >= arr.length)
            return 0;
        if(arr[i] % 3 != 0)
            return 1 + solve(arr , i+1);
        else
            return solve(arr , i+1);
    }
    public int minimumOperations(int[] nums) {
        return solve(nums , 0);
    }
}